contracts/pallet-ibc/src/ics20_fee/mod.rs

```diff
@@ -418,7 +418,10 @@ where
 			packet.data = serde_json::to_vec(&packet_data).map_err(|_| {
 				Ics04Error::implementation_specific("Failed to receiver account".to_string())
 			})?;
-			Pallet::<T>::deposit_event(Event::<T>::IbcTransferFeeCollected { amount: fee.into() })
+			Pallet::<T>::deposit_event(Event::<T>::IbcTransferFeeCollected {
+				amount: fee.into(),
+				asset_id,
```

Comment:
assset_id result

Reply:
should we even allow to go further if asset_id was an err. i think we should throw and error